### PR TITLE
Slicer

### PR DESCRIPTION
**Pull request by odinuv**
https://keboola.atlassian.net/browse/GCP-452

### CODE REVIEW

File: libs/slicer/src/Slicer.php [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        var_dump($os);
+        var_dump($platform);
+        $pwd = getcwd();
+        var_dump($pwd);
+        $pwd .= '/bin';
+        var_dump($pwd);
+        if (!file_exists($pwd)) {
+            mkdir($pwd);
+        }
+        $pwd .= '/slicer.exe';
```
Comment (ErikZigo) [inline libs/slicer/src/Slicer.php:45]: nemuselo by se to vzdycky stahovat jako slicer.exe :)

**Review by ErikZigo — COMMENTED** [review]
je to pekne, jen bude potreba jeste pridat `chmod +x`

**Comment by ErikZigo** [comment]
Asi se muze zavrit, pokracujeme tady https://github.com/keboola/platform-libraries/pull/181

**Comment by odinuv** [comment]
superseded by https://github.com/keboola/platform-libraries/pull/181

**Pull request closed without merging**